LICENSE

```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of elm-string-extra nor the names of its
+* Neither the name of elm-stylesheets nor the names of its
```

Comment:
Ha! Good catch. It was wrong before, and this would have changed it to a different wrong thing. 😅 

Fixed in 32832c2788033be75342e44f20aa8c92676c0721